templates/mail/issue/default.tmpl

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+	<title>{{.Subject}}</title>
+</head>
+
+<body>
+	{{if .IsMention}}<p>@{{.Doer.Name}} mentioned you:</p>{{end}}
+	<p>
+		{{- if eq .Body ""}}
+			{{if eq .ActionName "new"}}
+				Created #{{.Issue.Index}}.
+			{{else if eq .ActionName "close"}}
+				Closed #{{.Issue.Index}}.
+			{{else if eq .ActionName "reopen"}}
+				Reopened #{{.Issue.Index}}.
+			{{else}}
+				Empty comment on #{{.Issue.Index}}.
+			{{end}}
+		{{else}}
+			{{.Body | Str2html}}
+		{{end -}}
+	</p>
+	<p>
+		---
+		<br>
+		<a href="{{.Link}}">View it on Gitea</a>.
```

Comment:
How about change `Gitea` to the {{.AppName}}.

Reply:
Sure. I'll push the change after I have more feedback to avoid monopolizing the CI.

Reply:
@lunny done.